Tests/SonarQube.MSBuild.Tasks.UnitTests/SonarQube.MSBuild.Tasks.UnitTests.csproj

```diff
@@ -46,10 +46,6 @@
       <HintPath>..\..\packages\Microsoft.Build.Framework.14.3.0\lib\net45\Microsoft.Build.Framework.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="Microsoft.Build.Tasks.Core, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```

Comment:
Why is it removed?

Reply:
They were not needed - shouldn't have added them in the first place.